src/main/kotlin/io/streammachine/api/cli/commands/KafkaExporters.kt

```diff
@@ -0,0 +1,65 @@
+package io.streammachine.api.cli.commands
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.core.subcommands
+import com.github.ajalt.clikt.parameters.arguments.argument
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.choice
+import com.github.ajalt.clikt.parameters.types.enum
+import com.github.ajalt.clikt.parameters.types.int
+import com.github.ajalt.clikt.parameters.types.restrictTo
+import com.github.kittinunf.fuel.gson.jsonBody
+import com.github.kittinunf.fuel.httpDelete
+import com.github.kittinunf.fuel.httpGet
+import com.github.kittinunf.fuel.httpPut
+import io.streammachine.api.cli.common.ExporterCreateRequest
+import io.streammachine.api.cli.common.ExporterType
+import io.streammachine.api.cli.common.SinkType
+import io.streammachine.api.cli.common.printResponse
+
+class KafkaExporters : CliktCommand(
+    name = "kafka-exporters",
+    help = "Kafka exporters and details."
+) {
+    init {
+        subcommands(
+            KafkaExportersGet(),
+            KafkaExportersCreate(),
+            KafkaExportersDelete()
+        )
+    }
+
+    override fun run() = Unit
+}
+
+open class KafkaExportersGet : CliktCommand(
+    name = "get",
```

Comment:
We use the verb `list` elsewhere in the CLI. Though that implies getting a list, not a single exporter. Knowing that we'll create a version 2 soon for the CLI, do you agree to leave it like this?

Reply:
get indeed gets 0 or 1, so get seems more appropriate